### ISSUE TITLE
sbt-auto-build blows up if you fork to a different github user

### ISSUE THREAD

**Issue by DougC** [hmrc/sbt-auto-build#4]
I forked sbt-distributables to try and provide a pull request for an issue I raised, but sbt fails to load. It throws an exception complaining about the URL in the git config for `remote`. I've had a quick look but can't see immediately why. I've tried with two different github users with the same effect. 

```
[info] The config section 'remote' with subsection 'origin' had a url of 'git@github.com:WellFactored/sbt-distributables.git'
java.net.MalformedURLException: For input string: "wellfactored"
    at java.net.URL.<init>(URL.java:620)
    at java.net.URL.<init>(URL.java:483)
    at java.net.URL.<init>(URL.java:432)
    at sbt.package$.url(package.scala:25)
    at uk.gov.hmrc.ArtefactDescription$$anonfun$buildScmInfo$1$$anonfun$apply$15.apply(SbtAutoBuildPlugin.scala:140)
    at uk.gov.hmrc.ArtefactDescription$$anonfun$buildScmInfo$1$$anonfun$apply$15.apply(SbtAutoBuildPlugin.scala:139)
    at scala.Option.map(Option.scala:145)
    at uk.gov.hmrc.ArtefactDescription$$anonfun$buildScmInfo$1.apply(SbtAutoBuildPlugin.scala:139)
    at uk.gov.hmrc.ArtefactDescription$$anonfun$buildScmInfo$1.apply(SbtAutoBuildPlugin.scala:138)
    at scala.Option.flatMap(Option.scala:170)
    at uk.gov.hmrc.ArtefactDescription$.buildScmInfo(SbtAutoBuildPlugin.scala:138)
    at uk.gov.hmrc.ArtefactDescription$$anonfun$apply$12.apply(SbtAutoBuildPlugin.scala:109)
    at uk.gov.hmrc.ArtefactDescription$$anonfun$apply$12.apply(SbtAutoBuildPlugin.scala:109)
    at sbt.Init$Value$$anonfun$apply$13.apply(Settings.scala:604)
    at sbt.EvaluateSettings$$anonfun$sbt$EvaluateSettings$$constant$1.apply(INode.scala:163)
    at sbt.EvaluateSettings$$anonfun$sbt$EvaluateSettings$$constant$1.apply(INode.scala:163)
    at sbt.EvaluateSettings$MixedNode.evaluate0(INode.scala:175)
    at sbt.EvaluateSettings$INode.evaluate(INode.scala:135)
    at sbt.EvaluateSettings$$anonfun$sbt$EvaluateSettings$$submitEvaluate$1.apply$mcV$sp(INode.scala:69)
    at sbt.EvaluateSettings.sbt$EvaluateSettings$$run0(INode.scala:78)
    at sbt.EvaluateSettings$$anon$3.run(INode.scala:74)
    at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
    at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
    at java.lang.Thread.run(Thread.java:745)
Caused by: java.lang.NumberFormatException: For input string: "wellfactored"
    at java.lang.NumberFormatException.forInputString(NumberFormatException.java:65)
    at java.lang.Integer.parseInt(Integer.java:580)
    at java.lang.Integer.parseInt(Integer.java:615)
    at java.net.URLStreamHandler.parseURL(URLStreamHandler.java:216)
    at java.net.URL.<init>(URL.java:615)
    at java.net.URL.<init>(URL.java:483)
    at java.net.URL.<init>(URL.java:432)
    at sbt.package$.url(package.scala:25)
    at uk.gov.hmrc.ArtefactDescription$$anonfun$buildScmInfo$1$$anonfun$apply$15.apply(SbtAutoBuildPlugin.scala:140)
    at uk.gov.hmrc.ArtefactDescription$$anonfun$buildScmInfo$1$$anonfun$apply$15.apply(SbtAutoBuildPlugin.scala:139)
    at scala.Option.map(Option.scala:145)
    at uk.gov.hmrc.ArtefactDescription$$anonfun$buildScmInfo$1.apply(SbtAutoBuildPlugin.scala:139)
    at uk.gov.hmrc.ArtefactDescription$$anonfun$buildScmInfo$1.apply(SbtAutoBuildPlugin.scala:138)
    at scala.Option.flatMap(Option.scala:170)
    at uk.gov.hmrc.ArtefactDescription$.buildScmInfo(SbtAutoBuildPlugin.scala:138)
    at uk.gov.hmrc.ArtefactDescription$$anonfun$apply$12.apply(SbtAutoBuildPlugin.scala:109)
    at uk.gov.hmrc.ArtefactDescription$$anonfun$apply$12.apply(SbtAutoBuildPlugin.scala:109)
    at sbt.Init$Value$$anonfun$apply$13.apply(Settings.scala:604)
    at sbt.EvaluateSettings$$anonfun$sbt$EvaluateSettings$$constant$1.apply(INode.scala:163)
    at sbt.EvaluateSettings$$anonfun$sbt$EvaluateSettings$$constant$1.apply(INode.scala:163)
    at sbt.EvaluateSettings$MixedNode.evaluate0(INode.scala:175)
    at sbt.EvaluateSettings$INode.evaluate(INode.scala:135)
    at sbt.EvaluateSettings$$anonfun$sbt$EvaluateSettings$$submitEvaluate$1.apply$mcV$sp(INode.scala:69)
    at sbt.EvaluateSettings.sbt$EvaluateSettings$$run0(INode.scala:78)
    at sbt.EvaluateSettings$$anon$3.run(INode.scala:74)
    at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
    at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
    at java.lang.Thread.run(Thread.java:745)
```

**Comment by RawToast** [comment]
+1 Getting the same issue with this library. It always fails to import when cloned from a fork, but will work if you clone master.

**Comment by lesjoneshmrc** [comment]
Isn't this an issue with sbt-distributables - which currently depends on v0.0.8 - should be upgraded to v1.0.0

**Comment by lesjoneshmrc** [comment]
This should be fixed now via https://github.com/hmrc/sbt-distributables/issues/6

**Comment by ollyw** [comment]
I believe this issue will be fixed by the changes in #8 once merged
